lib/collection/variable.js

```diff
@@ -257,6 +257,28 @@ _.assign(Variable, /** @lends Variable */ {
          */
         number: Number,
 
+        /**
```

Comment:
This block is not a change, but merely a code reorder for clean addition of new type